### PR TITLE
add jb-everything 0.1.0 from pypi

### PR DESCRIPTION
**Pull request by jeromebaum**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jeromebaum** [comment]
I am willing to be listed as maintainer.

File: recipes/jb-everything/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/jb-everything/
+  summary: Large list of imports for machine learning.
+  license: MIT
+  license_file: LICENSE  # to be upstreamed
```
Comment (synapticarbors) [inline recipes/jb-everything/meta.yaml:51]: I opened a PR (https://github.com/jeromebaum/jb-everything/pull/1) to fix the need to manually include the license file here. Once it is in the source distribution, be sure to remove the local copy. I often recommend putting a comment in the `meta.yaml` as a reminder along the lines of:

```yaml
# License file manually packaged. See https://github.com/jeromebaum/jb-everything/pull/1
license_file: LICENSE
```

as a visual reminder for future updates. Also note that upstream the file is `LICENSE.txt` not `LICENSE`

File: recipes/jb-everything/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  skip: True  # [win]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - dabl ~=0.1.9
+    - dask[complete] ~=2.30.0  # to be upstreamed
```
Comment (synapticarbors) [inline recipes/jb-everything/meta.yaml:24]: I don't believe we support extras notation. You should just list the extra requirements explicitly.

File: recipes/jb-everything/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jb-everything-{{ version }}.tar.gz
+  sha256: efe53c547aae8794372817182db86822e039349ad6099ce4c9f0cd2e37f1c21a
+
+build:
+  number: 0
+  skip: True  # [win]
```
Comment (synapticarbors) [inline recipes/jb-everything/meta.yaml:15]: Is there a reason to skip windows and not just use `noarch: python`?

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

I'm not sure if every dependency is available on windows, but if they are, we should strive to make thie `noarch`.

File: recipes/jb-everything/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    - dabl ~=0.1.9
+    - dask[complete] ~=2.30.0  # to be upstreamed
+    - eli5 ~=0.10.1
+    - jax ~=0.2.7
+    - lightgbm ~=3.1.0
+    - matplotlib-base ~=3.3.3
+    - missingno ~=0.4.2
+    - numpy ~=1.19.4
+    - pandas ~=1.1.4
+    - python >=3.5  # to be upstreamed
```
Comment (synapticarbors) [inline recipes/jb-everything/meta.yaml:32]: If this is not a `noarch: python` recipe, then the restrictions on the python version should be up in the `skip` statement above.

File: recipes/jb-everything/meta.yaml (hunk 3 above)
Comment (jeromebaum) [reply recipes/jb-everything/meta.yaml:15]: Not every dependency is available on Windows - I've added a comment to that effect.

**Comment by jeromebaum** [comment]
@synapticarbors I think I've addressed all of the points (other than the recommendation about the license file comment - shouldn't be long to get that addressed in the upstream package).

**Comment by jeromebaum** [comment]
@synapticarbors I also fixed the license comment in line with your suggestion. Do you mind taking another look? Appreciate the holiday season is upon us so no worries if not. Happy holidays! :)
